data/soft-nep.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "SoftNEP",
+  "establishedAt": 1508298452,
+  "location": "Bhimsen Gola Marg, Min Bhawan, Kathmandu",
+  "website": "https://www.softnep.com",
+  "email": "mkt@softnep.com",
```

Comment:
Let's add enquiry email rather than marketing email for companies : `info@softnep.com`. 
They have career handling email :`career@softnep.com` as well but they may not respond to other enquiries other than taking CVs. 
![image](https://user-images.githubusercontent.com/66344649/201951160-6c3ab75c-df4b-4073-85df-2f7708bf5ce0.png)
Thoughts @thenishantsapkota ?

Reply:
Yeah all the emails should be enquiry emails!